test/features/support/GithubAPI/Authentication/AuthenticationTestSteps.js

```diff
@@ -0,0 +1,29 @@
+const agent = require('superagent');
+const { Given, When, Then } = require('cucumber');
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'manuelq12';
+const repository = 'workshop-api-cucumber';
+
+Given('a Github username', () => {
```

Comment:
why do you need an empty Given?

Reply:
Since I already had the username and repository in the constants, there's nothing I need to do there, unless you want me to change it to use examples